resen-core/resources/helpers/setup_citationhelper.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Could we not just add the `pip install git+` lines of code to the existing setup_py27 and setup_py36 scripts instead?

Reply:
Sure.  If we do that, where should setting the `CITEHELP_REFILE` environment variable go?

Reply:
Sure.  If we do that, where should setting the `CITEHELP_REFFILE` environment variable go?

Reply:
Environment variables are currently set starting on line 119 of the Dockerfile. See `TERM=xterm-256color`

Reply:
This has been changed so that the pip installation occurs in the two environment setup scripts and the environment variable is set on line 123.